hs_core/templates/pages/create-resource.html

```diff
@@ -3,153 +3,648 @@
                     </div>
+
+                    <ul class="dropdown-menu shadow-md" aria-labelledby="select-resource-type">
+                        <li>
+                            <a data-toggle="tooltip" data-placement="right"
+                               title="A resource that holds a list of other resources in HydroShare."
+                               data-value="CollectionResource">
+                                <img src="{{ STATIC_URL }}img/resource-icons/collection48x48.png"
+                                     alt="Collection Resource Icon" class="resource-type-icon"/>
+                                <div class="resource-type-name">&nbsp;Collection</div>
+                            </a>
+                        </li>
+                        <li>
+                            <a data-toggle="tooltip" data-placement="right"
+                               title="A resource which can include multiple files without file format limitations."
+                               data-value="CompositeResource">
+                                <img src="{{ STATIC_URL }}img/resource-icons/composite48x48.png"
+                                     alt="Composite Resource Icon" class="resource-type-icon"/>
+                                <div class="resource-type-name">&nbsp;Composite</div>
+                            </a>
+                        </li>
+                        <li>
+                            <a data-toggle="tooltip" data-placement="right"
+                               title="An arbitrary file or collection of files that are logically grouped together as a resource."
+                               data-value="GenericResource">
+                                <img src="{{ STATIC_URL }}img/resource-icons/generic48x48.png"
+                                     alt="Generic Resource Icon" class="resource-type-icon"/>
+                                <div class="resource-type-name">&nbsp;Generic</div>
+                            </a>
+                        </li>
+                        <li>
+                            <a data-toggle="tooltip" data-placement="right"
+                               title="A shortcut (or reference) to an external web application that functions with HydroShare data resources."
+                               data-value="ToolResource">
+                                <img src="{{ STATIC_URL }}img/resource-icons/webapp48x48.png"
+                                     alt="Web App Resource Icon" class="resource-type-icon"/>
+                                <div class="resource-type-name">&nbsp;Web App</div>
+                            </a>
+                        </li>
+                        <li role="separator" class="divider"></li>
+                        <li>
+                            <a data-toggle="tooltip" data-placement="right"
+                               title="Defines the input and output data for a generic hydrological model, for a specific time and place."

```

Comment:
@mauriel133 Would it be better to make the tooltip text a bit clearer by changing it to ```a model instance resource that defines the input and ...```? Just a suggestion.